azurerm/internal/services/network/express_route_circuit_peering_resource.go

```diff
@@ -108,6 +108,54 @@ func resourceArmExpressRouteCircuitPeering() *schema.Resource {
 				},
 			},
 
+			"ipv6_peering": {
```

Comment:
given the resource name i think peering is implied?
```suggestion
			"ipv6": {
```